packages/schema/build.config.ts

```diff
@@ -9,7 +9,7 @@ export default defineBuildConfig({
       name: 'config',
       builder: 'untyped',
       defaults: {
-        rootDir: '/project/'
+        rootDir: '/path/to/your/app/'
```

Comment:
What about something like this and instead properly document dirs definition (`<rootDir>` is documented)

```suggestion
        rootDir: '/<rootDir>/'
```